docfx_project/ziti/overview.md

```diff
@@ -5,9 +5,60 @@ uid: zitiOverview
 # Overview
 
 ## Why Ziti?
-Ziti represents the next generation of secure networking for your applications and has arrived with the Ziti platform,
-created by NetFoundry, Inc ! The Ziti Controller and the Ziti Edge Router combine to form a secure, Zero Trust
-entry point into your network or as a secure zero-trust proxy to other hosted services.
+Ziti represents the next generation of secure, open-source networking for your applications and has arrived with the Ziti platform, created by NetFoundry, Inc ! The Ziti Controller and the Ziti Edge Router combine to form a secure, Zero Trust entry point into your network or as a secure zero-trust proxy to other hosted services.
+
+Ziti provides several security features, including:
+
+* Zero Trust and Service Segmentation
```

Comment:
I've seen 'network segmentation' and 'application segmentation' before. Application Segmentation really highlights Ziti's SDK approach.

Reply:
I had application segmentation but switched it to service segmentation to try and be consistent since I had a mix of application and service. I could try changing all the service -> application and see if it still makes sense. Or maybe I can equate applications and services at some point and see if I can make a smooth transition.